### PR TITLE
hsigmoid loss docs

### PR DESCRIPTION
**Pull request by zhupengyang**
cn_0
cn_1

### CODE REVIEW

File: doc/paddle/api/paddle/nn/functional/loss/hsigmoid_loss_cn.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
+hsigmoid
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/nn/functional/loss/hsigmoid_loss_cn.rst]: 第一行为文件的lable， .. _cn_path_to_api:
hsigmoid -> hsigmoid_loss

File: doc/paddle/api/paddle/nn/functional/loss/hsigmoid_loss_cn.rst [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+-------------------------------
+
+.. py:function:: paddle.nn.functional.hsigmoid_loss(input, label, num_classes, weight, bias=None, path_table=None, path_code=None, is_sparse=False, name=None)
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/nn/functional/loss/hsigmoid_loss_cn.rst:6]: weight 还是 weight_attr?
bias 还是 bias_attr?

File: doc/paddle/api/paddle/nn/functional/loss/hsigmoid_loss_cn.rst [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+3. 建立一个dict类型数据结构，用于存储 **单词id -> 该单词叶结点至根节点路径的编码** 的映射，即路径编码 ``path_code`` 参数。 编码是指每次二分类的标签，1为真，0为假；
+
+4. 每个单词都已经有自己的路径和路径编码，当对于同一批输入进行操作时，可以同时传入一批路径和路径编码进行运算。
+
+参数
+::::::::::
+    - **input** (Tensor) - 输入Tensor。数据类型为float32或float64，形状为 ``[N, D]`` ，其中 ``N`` 为minibatch的大小，``D`` 为特征大小。
+    - **label** (Tensor) - 训练数据的标签。数据类型为int64，形状为 ``[N, 1]`` 。
+    - **num_classes** (int) - 类别总数(字典大小)必须大于等于2。若使用默认树结构，即当 ``path_table`` 和 ``path_code`` 都为None时 ，必须设置该参数。若使用自定义树结构，即当 ``path_table`` 和 ``path_code`` 都不为None时，它取值应为自定义树结构的非叶节点的个数，用于指定二分类的类别总数。
+    - **weight_attr** (Tensor) - 该OP的权重参数。形状为 ``[numclasses-1, D]`` ，数据类型和 ``input`` 相同。
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/nn/functional/loss/hsigmoid_loss_cn.rst]: 同上

File: doc/paddle/api/paddle/nn/functional/loss/hsigmoid_loss_cn.rst [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    - **bias_attr** (Tensor, 可选) - 该OP的偏置参数。形状为 ``[numclasses-1, 1]`` ，数据类型和 ``input`` 相同。如果设置为None，将没有偏置参数。默认值为None。
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/nn/functional/loss/hsigmoid_loss_cn.rst]: 同上

File: doc/paddle/api/paddle/nn/layer/loss/HSigmoidLoss_cn.rst [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+3. 建立一个dict类型数据结构，用于存储 **单词id -> 该单词叶结点至根节点路径的编码** 的映射，即路径编码 ``path_code`` 参数。 编码是指每次二分类的标签，1为真，0为假；
+
+4. 每个单词都已经有自己的路径和路径编码，当对于同一批输入进行操作时，可以同时传入一批路径和路径编码进行运算。
+
+参数
+::::::::::
+    - **feature_size** (int) - 输入Tensor的特征大尺寸。
+    - **num_classes** (int) - 类别总数(字典大小)必须大于等于2。若使用默认树结构，即当 ``is_custom=False`` 时 ，必须设置该参数。若使用自定义树结构，即当 ``is_custom=True`` 时，它取值应为自定义树结构的非叶节点的个数，用于指定二分类的类别总数。
+    - **param_attr** (ParamAttr, 可选) – 指定权重参数属性的对象。默认值为None，表示使用默认的权重参数属性。具体用法请参见 :ref:`cn_api_fluid_ParamAttr` 。
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/nn/layer/loss/HSigmoidLoss_cn.rst]: weight_attr

File: doc/paddle/api/paddle/nn/layer/loss/HSigmoidLoss_cn.rst [hunk 6]
```diff
@@ -0,0 +1,55 @@
+HSigmoidLoss
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/nn/layer/loss/HSigmoidLoss_cn.rst:3]: 同上 第一行为API文档label

File: doc/paddle/api/paddle/nn/functional/loss/hsigmoid_loss_cn.rst (hunk 1 above)
Comment (zhupengyang) [reply doc/paddle/api/paddle/nn/functional/loss/hsigmoid_loss_cn.rst]: Done

File: doc/paddle/api/paddle/nn/functional/loss/hsigmoid_loss_cn.rst (hunk 2 above)
Comment (zhupengyang) [reply doc/paddle/api/paddle/nn/functional/loss/hsigmoid_loss_cn.rst:6]: weight/bias

File: doc/paddle/api/paddle/nn/functional/loss/hsigmoid_loss_cn.rst (hunk 3 above)
Comment (zhupengyang) [reply doc/paddle/api/paddle/nn/functional/loss/hsigmoid_loss_cn.rst]: weight/bias

File: doc/paddle/api/paddle/nn/functional/loss/hsigmoid_loss_cn.rst (hunk 4 above)
Comment (zhupengyang) [reply doc/paddle/api/paddle/nn/functional/loss/hsigmoid_loss_cn.rst]: weight/bias

File: doc/paddle/api/paddle/nn/layer/loss/HSigmoidLoss_cn.rst (hunk 5 above)
Comment (zhupengyang) [reply doc/paddle/api/paddle/nn/layer/loss/HSigmoidLoss_cn.rst]: Done

File: doc/paddle/api/paddle/nn/layer/loss/HSigmoidLoss_cn.rst (hunk 6 above)
Comment (zhupengyang) [reply doc/paddle/api/paddle/nn/layer/loss/HSigmoidLoss_cn.rst:3]: Done

**Review by TCChenlong — APPROVED** [review]
LGTM

**Review by hong19860320 — APPROVED** [review]
LGTM
